compiler/damlc/daml-stdlib-src/DA/Internal/Template/Functions.daml

```diff
@@ -62,16 +62,12 @@ class HasSoftExercise t c r | t c -> r where
   -- | HIDE
   _softExercise : ContractId t -> c -> Update r
 
-#ifdef DAML_PACKAGE_UPGRADES
-
 softFetch : HasSoftFetch t => ContractId t -> Update t
 softFetch = _softFetch
 
 softExercise : HasSoftExercise t c r => ContractId t -> c -> Update r
 softExercise = _softExercise
 
-#endif -- DAML_PACKAGE_UPGRADES
-
```

Comment:
these removed lines are responsible for the test failures in `//daml-script/test:test_test_suite_src_com_digitalasset_daml_lf_engine_script_test_Daml{2,3}ScriptTestRunner.scala`, since those expect an error with 

```
Details: Last location: [DA.Internal.Template.Functions:242]
```

but now the error mentions line `247` instead. I think the line number goes up because that codepath had `DAML_PACKAGE_UPGRADES` undefined so the lines that define `softFetch` and `softExercise` (plus the blank line in between) weren't there

Reply:
that is to say, the fix is to replace `DA.Internal.Template.Functions:242` with `DA.Internal.Template.Functions:247` in `daml-script/test/src/com/digitalasset/daml/lf/engine/script/test/Daml2ScriptTestRunner.scala` and `daml-script/test/src/com/digitalasset/daml/lf/engine/script/test/Daml3ScriptTestRunner.scala`

Reply:
Thanks for chasing the line number!